app/src/main/java/com/example/lapnguyen/weather_3/screen/mainscreen/MainPresenter.java

```diff
@@ -0,0 +1,64 @@
+package com.example.lapnguyen.weather_3.screen.mainscreen;
+
+import android.Manifest;
```

Comment:
ERROR: Unused import: android.Manifest. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports